idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/solid_phase_thermo.py

```diff
@@ -48,7 +48,7 @@
     degrees_of_freedom,
     number_unfixed_variables_in_activated_equalities)
 import idaes.logger as idaeslog
-from idaes.core.util import get_solver
+from idaes.core.util import get_solver, scaling as iscale
 
```

Comment:
The suggested changes to the gas property package also apply to the solid property package.